Batch documentation (#7677)
segmentio/segment-docs: src/connections/destinations/catalog/batch/index.md

---
title: Batch Destination

> id: 596d11f870a3e552b957e6d9

Reviewer: Was the page ID deliberately deleted?

{% include content/plan-grid.md name="actions" %}   

> Batch is a powerful customer engagement platform that helps businesses deliver personalized, timely notifications and messages to boost user retention and drive growth. Discover how Batch can transform your communication strategy at Batch.

Reviewer: ```suggestion
Batch is a powerful customer engagement platform that helps businesses deliver personalized, timely notifications and messages to boost user retention and drive growth. For details on how Batch can transform your communication strategy, see [Batch.com](https://batch.com/platform){:target=_"blank"}.
```

Minor rewording

This destination is maintained by Batch. For any issues with the destination, [contact their Support team](mailto:support@batch.com).

## Getting started

> 1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="_blank"} search for Batch.

Reviewer: ```suggestion
1. From your workspace's [destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="_blank"} search for "Batch".
```
Amended capitalization

2. Select **Batch** and click **Add Destination**.

> 3. Select an existing Source to connect to Batch (Actions).

Reviewer: ```suggestion
3. Select an existing source to connect to Batch (Actions).
```
Amended capitalization
